Flash Chat iOS13/Controllers/ThreadListViewController.swift

```diff
@@ -15,13 +15,15 @@ class ThreadListViewController: UITableViewController {
 
 	let database = Firestore.firestore()
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		navigationController?.navigationItem.hidesBackButton = true
```

Comment:
⚒️ 

If this isn't working yet, let's revert this code change

Reply:
Fixed